addons/sourcemod/scripting/SprayManager.sp

```diff
@@ -2731,27 +2732,22 @@ bool SprayBanClient(int client, int target, int iBanLength, const char[] sReason
 
 bool SprayUnbanClient(int target, int client=-1)
```

Comment:
if client is -1, all replycommand will error out no ?

Reply:
It should be good now.

Reply:
no because this function is used like so here:
- https://github.com/Rushaway/sm-plugin-SprayManager/blob/df91ce3845cbdc4fe9a3774725b10f72f4029a74/addons/sourcemod/scripting/SprayManager.sp#L2186

you only need to avoid the replycommand stuff, not the entire function